easybuild/easyblocks/p/pytorch.py

```diff
@@ -339,6 +355,38 @@ def configure_step(self):
 
         pytorch_version = LooseVersion(self.version)
 
+        self.has_xml_test_reports = False
+        if pytorch_version >= '1.10.0':
+            out, ec = run_cmd(self.python_cmd + " -c 'import xmlrunner'", log_ok=False)
```

Comment:
Hmm, why not just check that unittest-xml-reporting is a builddep?

Reply:
Because I wanted to anticipate that we might have it in Python or Python-bundle-PyPI

Reply:
I can imagine others need that too, even users and we provide e.g. pytest too. and the one additional call doesn't hurt in the testing of PyTorch ;-)